source/form.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона | Форма отзыва</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html">
+        <img src="img/city-of-sedona-logo.svg" width="185" height="96" alt="Логотип города Седона.">
+      </a>
+
+      <nav class="main-header__nav">
+        <button class="nav__toggle" type="button">
+          <span class="visually-hidden">Меню.</span>
+        </button>
+        <div class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="index.html">Главная</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="catalog.html">Фото и видео</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link nav-menu__link--current" href="form.html">Форма отзыва</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="https://htmlacademy.ru">HTML Academy</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main-container">
+      <h1 class="visually-hidden">Форма отзыва о туристическом городе Sedona.</h1>
+
+      <section class="background-image">
+        <img class="sedona-logo" src="img/sedona-logo.svg" width="229" height="39" alt="Седона.">
+      </section>
+
+      <section class="feedback">
+        <h2 class="feedback__title">Оставьте свой отзыв</h2>
+        <p class="feedback__text">Помогите нашим отелям стать лучше! Оставьте отзыв о них, а также о посещенных вами достопримечательностях</p>
+        <form class="feedback__form" action="https://echo.htmlacademy.ru" method="post">
+          <fieldset class="feedback__item feedback__item--name">
+            <legend class="feedback__legend">Представьтесь:</legend>
+            <label class="feedback__field" for="name">Имя*:
+              <input class="field__input" type="text" name="name" id="name" placeholder="Пётр" required>
+            </label>
+            <label class="feedback__field" for="surname">Фамилия*:
+              <input class="field__input" type="text" name="surname" id="surname" placeholder="Пустота" required>
+            </label>
+            <label class="feedback__field" for="middle-name">Отчество:
+              <input class="field__input" type="text" name="middle-name" id="middle-name" placeholder="Викторович">
```

Comment:
Думаю здесь лучше так чтобы не париться с блоком
```suggestion
              <input class="feedback__input" type="text" name="middle-name" id="middle-name" placeholder="Викторович">
```